spec/integration/user_show_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+RSpec.describe 'User show page', type: :feature do
+  before(:each) do
+    @user1 = User.create(name: 'Tom',
+                         photo: 'https://cdn.pixabay.com/photo/2015/06/22/08/40/child-817373_1280.jpg', bio: 'Tom bio')
+    @post1 = Post.create(author_id: @user1.id, title: 'Hello', text: 'This is my first post')
+    @post2 = Post.create(author_id: @user1.id, title: 'Hi', text: 'This is my second post')
+
+    visit user_path(@user1)
+  end
+  describe 'Display a specific user details and posts' do
+    it 'Should display the name of the user' do
+      expect(page).to have_content(@user1.name)
+    end
+    it 'Should display the user photo' do
+      expect(page).to have_css(
+        "img[src*='https://cdn.pixabay.com/photo/2015/06/22/08/40/child-817373_1280.jpg']", visible: :visible
+      )
+    end
+    it 'Should display user post counter' do
+      expect(page).to have_content("Number of posts: #{@user1.posts.count}")
+    end
+    it 'Should display the bio of the user' do
+      expect(page).to have_content(@user1.bio)
+    end
+    it 'Should display the first 2 posts' do
+      expect(page).to have_content(@post1.text)
+      expect(page).to have_content(@post2.text)
+    end
```

Comment:
- [ ] Good job adding tests 👍, however, let's display the first 3 posts instead of displaying only 2 as it is [required](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/intergration_spec_and_n_1_project.md#:~:text=I%20can%20see%20the%20user%27s%20first%203%20posts.) 👍. Kindly, fix it 😄 

![Screenshot 2023-09-07 191443](https://github.com/sylvester-wamaya/blog-app/assets/57408419/9782bc8a-7007-470d-8d5c-02443890be77)